components/pkg-export-helm/src/deps.rs

```diff
@@ -24,8 +24,8 @@ use export_docker::Result;
 use error::Error;
 
 pub const DEFAULT_OPERATOR_VERSION: &'static str = "0.5.1";
-pub const OPERATOR_REPO_URL: &'static str = "https://kinvolk.github.io/habitat-operator\
-                                             /helm/charts/stable/";
+pub const OPERATOR_REPO_URL: &'static str = "https://habitat-sh.github.io/\
```

Comment:
Out of curiosity, why is this line broken?

Reply:
well to fit in one line (100 chars) but chosen to be broken at a point where it's more readable IMO.

Reply:
which part is weird? In code, you are not supposed to go beyond the character limit and rustfmt check will break if you do.